views/includes/global-header.njk

```diff
@@ -38,10 +38,10 @@
             <div class="global-header__content-primary">
                 <a href="/" class="global-header__logo brand-logo">
                     <span class="u-visually-hidden">
-                        {{ __("global.brand.title") }}
```

Comment:
This was causing the descriptive text for the logo to still read "Big Lottery Fund"